### PR TITLE
feat(Shadow): update shadow TS definiton to latest structure

### PR DESCRIPTION
**Pull request by THoj13**
## Description
Add updated TypeScript definition for Shadow component.

## References
LUI-966

## Testing
n/a

## Automation
n/a

## Checklist

- [x] all commented code has been removed
- [x] any new console issues have been resolved
- [x] code linter and formatter has been run
- [x] test coverage meets repo requirements
- [x] PR name matches the expected semantic-commit syntax

### CODE REVIEW

File: packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts [hunk 1]
```diff
@@ -20,8 +20,9 @@ import type lng from '@lightningjs/core';
 import Base from '../Base';
 import type { Color, StylePartial } from '../../types/lui';
```
Comment (soumyaloka) [inline packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts]: type should be removed

**Review by soumyaloka — APPROVED** [review]

File: packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts [hunk 2]
```diff
@@ -20,8 +20,9 @@ import type lng from '@lightningjs/core';
...
 
-export type ShadowStyle = {
+type ShadowStyle = {
   alpha: number;
+  animation: lng.types.TransitionSettings.Literal;
```
Comment (anthony9187) [inline packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts:25]: is there a different type for animations, or are we using the word animation to mean transition here?

Comment (THoj13) [reply packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts:25]: The `animation` style prop is actually referring to a transition: `theme.animation.standardEntrance`. 
```js
this.applySmooth(
  this._Shadow,
  { alpha: this.style.alpha, y: this.style.offsetY, x: this.style.offsetX },
  {
    alpha: [this.style.alpha, this.style.animation],
    y: [this.style.offsetY, this.style.animation],
    x: [this.style.offsetX, this.style.animation]
  }
);
```

File: packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts (hunk 1 above)
Comment (THoj13) [reply packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts]: I believe this works with `import` or `import type`, but can remove it to be consistent with other TS def files in the repo.

**Review by anthony9187 — APPROVED** [review]

**Pull request merged**